### PR TITLE
Mount db in a seperate folder

### PR DESCRIPTION
**Pull request by pingiun**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN mkdir /eetfestijn && \
     mkdir -p /eetfestijn/log/ && \
     touch /eetfestijn/log/uwsgi.log && \
     chown -R www-data:www-data /eetfestijn && \
-    mkdir -p /usr/src/app/
+    mkdir -p /usr/src/app/
```
Comment (thomwiggers) [inline Dockerfile:25]: We don't need to add this space here.

File: eetfestijn/settings.py [hunk 2]
```diff
@@ -80,7 +80,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'NAME': os.path.join(BASE_DIR, 'db', 'db.sqlite3'),
```
Comment (thomwiggers) [inline eetfestijn/settings.py:83]: I'm assuming this is to make writing a dockerfile easier, but could you clarify that in the PR description?

Comment (thomwiggers) [reply eetfestijn/settings.py:83]: ah too late

File: Dockerfile (hunk 1 above)
Comment (joren485) [reply Dockerfile:25]: Fixed in #10

**Pull request merged**